src/index.html

```diff
@@ -1,12 +1,346 @@
+<!-- #region head -->
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Moyo header</title>
+    <link rel="stylesheet" href="styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <!-- #endregion -->
+
+  <body class="page__body">
+
+    <header class="header">
+      <div class="header__logo">
+        <a href="index.html" id="home">
+          <img src="images/logo.png" alt="MOYO Logo">
+        </a>
+      </div>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__item">
```

Comment:
But don't forget to add empty lines between multiline sibling blocks of HTML
```suggestion
          </li>
          
          <li class="nav__item">
```